blueprints/addon/files/.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
This file should be inherited from the app blueprint. I don't think we need it here.